### PR TITLE
Replace prism+chakra Code component with shiki+tailwind

### PR DESCRIPTION
**Pull request by MananTank**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR focuses on refactoring the code for handling code blocks in the UI, replacing instances of `CodeBlock` with `CodeClient` and `PlainTextCodeBlock`, and updating styles and imports accordingly. It also adds new functionalities related to code formatting.

### Detailed summary
- Removed `CodeBlock` components and replaced with `CodeClient` or `PlainTextCodeBlock`.
- Introduced `getCodeHtml` function to format code and convert it to HTML.
- Added new CSS styles for dark mode support in code rendering.
- Updated `package.json` to include `shiki` for syntax highlighting.
- Cleaned up imports across multiple components.

> The following files were skipped due to too many changes: `pnpm-lock.yaml`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by MananTank** [comment]
* **#5393** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5393?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @MananTank and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/typedoc-gen/package.json
Comment (jnsdls) [inline packages/typedoc-gen/package.json:1]: Oversight?

File: packages/unity-js-bridge/package.json
Comment (jnsdls) [inline packages/unity-js-bridge/package.json:1]: Oversight?

**Review by jnsdls — CHANGES_REQUESTED** [review]

File: packages/typedoc-gen/package.json
Comment (MananTank) [reply packages/typedoc-gen/package.json:1]: Ahh, this got generated when I ran `pnpm run lint` on root

Comment (MananTank) [reply packages/typedoc-gen/package.json:1]: removing it

Comment (MananTank) [reply packages/typedoc-gen/package.json:1]: removed

File: packages/unity-js-bridge/package.json
Comment (MananTank) [reply packages/unity-js-bridge/package.json:1]: removed

**Review by jnsdls — APPROVED** [review]

**Pull request merged**